components/profile_popover/profile_popover.jsx

```diff
@@ -461,7 +461,6 @@ class ProfilePopover extends React.PureComponent {
                         >
                             <ToggleModalButtonRedux
                                 accessibilityLabel={addToChannelMessage}
-                                ref='addUserToChannelModalButton'
```

Comment:
Was causing a runtime console error/warning, is not referenced anywhere, presumed safe to remove.